### DISCUSSION TITLE
Upgrading to Tasmota 13.x - Error with Partition Wizard

### DISCUSSION THREAD

**Discussion by SimonFili** [arendst/Tasmota#19780] [Q&A] [answered]
Hey guys!

I was confident... I did that before, it's going to be easy!

So I have an ESP32 with version 11.1. I compiled a new Tasmota32 with EZO chips and want to update.  I knew I needed the partition wizard. So uploaded, reboot, and ran it. But... I get this in the console:

```
14:06:11.239 BRY: Exception> 'attribute_error' - the 'Partition_info' object has no attribute 'sz'
14:06:11.246 stack traceback:
14:06:11.248 	<unknown source>: in function `show_current_partitions`
14:06:11.253 	<unknown source>: in function `page_part_mgr`
14:06:11.257 	<unknown source>: in function `<lambda>`
```
Anything I missed?

Thanks!  Simon

**Comment by sfromis** [comment, marked as answer]
Looks like your current Partition Wizard does not work with the pretty old Tasmota version, as some field names were changed. What I'd suggest would be to first upgrade to version 12.5, and then run the Partition Wizard there to repartition, before continuing to current version.
http://ota.tasmota.com/tasmota32/release-12.5.0/

**Reply by SimonFili** [reply]
Yep, that does it.  Running partition wizard now.  Thanks!

**Comment by SimonFili** [comment]
Going from 11.x to 13.x, you need partition wizard, but do do this, you need to install 12.5 before, so 2 steps process.
